### DISCUSSION TITLE
Extracting Old Crab/Parasite DNA

### DISCUSSION THREAD

**Discussion by aspencoyle** [RobertsLab/resources#1321] [Q&A]
I'm looking to try to extract DNA from archived samples to see if they're still viable. 

**Sample Description:**
- Date Collected: Ranges from 2003-2019
- Tissue: Almost all are hemolymph (blood) samples, with a few tissue samples mixed in
- Preservative: 100% EtOH
- Number of samples: Around 30,000, so losing a few isn't a concern
- Species: _Chionoecetes sp._ AKA snow and Tanner crab.  There are also a few samples of other crustacean species, including some king crab, hermit crab, and shrimp. However, the focus will likely be on the _Chionoecetes_ samples. 
- Infection status: Many of these samples are infected with the parasitic dinoflagellate _Hematodinium spp._ This parasite multiplies within the hemolymph and tissue of the host. We'd be interested in DNA from both the host AND parasite.

Any suggestions on specific methods and protocols for extraction? Tentative plan was to proceed using the Zymo EZ-DNA kit, but if others have better suggestions, would love to hear them!

**Comment by kubu4** [comment]
> to see if they're still viable

What are  you hoping to do with the DNA? The biggest potential concern here is the yields you'll get from hemolymph. Another concern would be DNA integrity, but as long as you're not planning on doing genome sequencing/assembly, this shouldn't be much of an issue.

 > Tentative plan was to proceed using the Zymo EZ-DNA kit

Should work just fine. Remove as much EtOH from the pelleted hemolymph before starting the protocol.

**Reply by sr320** [reply]
Agree - hardest thing I think will be having DNA - will need to pellet but not break cells.
